### PR TITLE
docs: Added contributing guideline and basic docs

### PR DESCRIPTION
**Pull request by utkarsh0908**
Added Docs for the project.
Kindly check the Getting Started part

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
-# tap
-The official website of Training and Placement Cell, IIIT Ranchi
+# House of Geeks - Official Technical Communtiy of IIITR
```
Comment (EmperorYP7) [inline README.md]:
```suggestion
# Training and Placement Cell, IIIT Ranchi Web App
```

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Just a few tweaks and we're good to go!

**Review by EmperorYP7 — APPROVED** [review]
LGTM

**Comment by utkarsh0908** [comment]
> Just a few tweaks and we're good to go!

Looking good to me.

**Pull request merged**